Corrfunc/tests/test_mocks.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+from os.path import dirname, abspath, join as pjoin
+
+import pytest
+import numpy as np
+
+from Corrfunc.tests.common import Mr19_mock_northonly, Mr19_randoms_northonly
+from Corrfunc.tests.common import (check_against_reference,
+                                   check_vpf_against_reference)
+from Corrfunc.tests.common import generate_isa_and_nthreads_combos
+    
+
+@pytest.mark.parametrize('isa,nthreads', generate_isa_and_nthreads_combos())
+def test_DDrppi_mocks(Mr19_mock_northonly, isa, nthreads):
```

Comment:
Whoa - is this the normal convention? `Mr19_mock_northonly` is both a function and a parameter? Particularly, on line 24, is there a function call involved or is `pytests` caching that function call return value?

Reply:
Yes! It's called a fixture: https://docs.pytest.org/en/6.2.x/fixture.html#back-to-fixtures